txtHistoryRust/README.md

```
@@ -28,6 +38,18 @@ The application uses a SQLite database with the following schema:
 - `service`: Service type (iMessage, SMS, etc.)
 - `thread_id`: Original thread ID
 - `has_attachments`: Flag indicating if the message has attachments
+- `contact_id`: Foreign key to contacts table
+
+### Processed Messages Table
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)